lib/quasar-config.js

```diff
@@ -404,6 +404,12 @@ class QuasarConfig {
           overwrite: true
         }
       }, cfg.electron || {}, {
+        builder: {
+          directories: {
+            app: appPaths.resolve.app(cfg.build.distDir),
```

Comment:
If quasar doesn't use 2 package.json layout (https://www.electron.build/tutorials/two-package-structure), and I guess it doesn't, `app` must be not used here. Instead, you should set `projectDir` (point to a dir where app package.json, node_modules and app code are located). 

`builder.build({ config: builderConfig, projectDir:  appPaths.resolve.app(cfg.build.distDir)})`

but projectDir is not required if project dir equals to current working directory.

Reply:
Thanks for the quick response! I've tested both approaches here, and concluded that the former is better.

Now I realize that an Electron-packaged Quasar Framework project **does** have a 2-`package.json` layout. In fact, when I run `quasar build -m electron` in my project, it firstly bundles my sources using Webpack, placing the results in directory e.g. `dist/electron-mat/UnPackaged/`, which has its own (automatically generated) `package.json`, and then invoke electron-builder, placing its results (packages, `linux-unpacked` directory, etc.) in directory `dist/electron-mat/`.

Reply:
In general, electron-builder is very powerful and such ugly (sorry, as author of electron-webpack I am sure that no need to use two-package.json layout) workarounds, that probably required for electron-packager, not needed for electron-builder. You can do whatever you want using `files`, for example, see electron-webpack default configuration (electron-builder is able to inherit configurations — `extends` like eslint) — https://github.com/electron-userland/electron-webpack/blob/master/packages/electron-webpack/electron-builder.yml As you can see, you can easily setup any wanted file mappings and reflect any complex project structure without need to use some stage dir and other such ugly workarounds.

I understand, that such refactoring maybe out of scope for this PR, but just to make sure you understand and you are aware of.